src/System.IO.Pipes/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,7 @@
+Compat issues with assembly System.IO.Pipes:
+TypesMustExist : Type 'System.IO.Pipes.PipeAccessRights' does not exist in the implementation but it does exist in the contract.
```

Comment:
@pjanotti can you please confirm these APIs are not in the reference assembly intentionally? I believe that is the case as they are exposed in System.IO.Pipes.AccessControl instead.

Reply:
Yes, the sources for those were moved to System.IO.Pipes but not the references since they were exposed in System.IO.Pipes.AccessControl in previous versions.